### DISCUSSION TITLE
Turing error smart screen Python on linux

### DISCUSSION THREAD

**Discussion by DJRoby19** [mathoudebine/turing-smart-screen-python#470] [General]
Hello everyone when I'm in the configure.py screen and I do save and run
It gives me this error on Linux Mint 21.3
root@Ryzen:/opt/turing-smart-screen-python-main# python3 configure.py
/usr/bin/env: "python": File o directory non esistente

and then when I open the terminal without root permissions to start it after I launch main.py it gives me this error
what can it be?

 roberto  /opt/turing-smart-screen-python-main  python3 main.py
16/02/2024 19:37:38 [INFO] Loading theme LandscapeBigLinux from res/themes/LandscapeBigLinux/theme.yaml
16/02/2024 19:37:38 [DEBUG] HW revision: A
16/02/2024 19:37:38 [DEBUG] Auto detected COM port: /dev/ttyACM0
16/02/2024 19:37:38 [ERROR] Cannot open COM port /dev/ttyACM0: [Errno 13] could not open port /dev/ttyACM0: [Errno 13] Permesso negato: '/dev/ttyACM0'
 roberto  /opt/turing-smart-screen-python-main 

**Comment by mathoudebine** [comment]
Hi, there is no need to run the program as root. You just need to add your user to the `dialout` group:
See here https://github.com/mathoudebine/turing-smart-screen-python/wiki/Troubleshooting#linux-cannot-open-serial-port-permission-error
